ui-tests/src/test/java/org/openmrs/reference/EditFormTest.java

```diff
@@ -32,33 +32,34 @@ public void setUp() throws Exception {
         patientDashboardPage = new ClinicianFacingPatientDashboardPage(page);
     }
 
-    @Ignore//ignore due to moving forms functionality
     @Test
     public void EditFormTest() throws Exception {
-        homePage.goToManageForm();
-        if (!manageForm.addPresent()) {
-            manageForm.delete();
-        }
-        manageForm.add();
-        manageForm.addLabel("Eye Report");
-        manageForm.addIcon("icon-align-justify");
-        manageForm.formIdFromUrl();
-        manageForm.save();
-        headerPage.clickOnHomeIcon();
-        homePage.goToManageForm();
-        manageForm.editPath();
-        manageForm.addLabel("Eye Test");
-        manageForm.save();
-        headerPage.clickOnHomeIcon();
-        homePage.goToActiveVisitPatient();
-        assertNotNull("Eye Test", patientDashboardPage.FORM_EXIST);
-//        Delete Form
-        headerPage.clickOnHomeIcon();
-        homePage.goToManageForm();
-        manageForm.deletePath();
-        assertNotNull("Add", manageForm.ADD);
+  	 homePage.goToManageForm();
+	 String Add = "containing add button";
+     if(!manageForm.containsText(Add)) {
+ 		headerPage.clickOnHomeIcon();
+     	assertNotNull("Eye Report","patientDashboardPage.FORM_EXIST");
+	}else {
```

Comment:
please format your code well to make your changes more readable and ensure all checks pass